src/lashup_kv.erl

```diff
@@ -48,10 +49,12 @@
 
 
 -include("lashup_kv.hrl").
--type kv() :: #kv{}.
+-type kv2() :: #kv2{}.
```

Comment:
It might make sense to reuse lashup:kv -> #kv2{}. Since lashup:kv() and lashup:kv2() aren't coexisting, I don't see a reason to keep them together.